AutomationHelpers/src/UserCodeCollections/FileLibrary.cs

```diff
@@ -152,9 +153,169 @@ public static void WaitForFile(string path, string pattern, int duration, int in
             }
         }
 
+        /// <summary>
+        /// Compares content of two binary files.
+        /// </summary>
+        /// <param name="filePath1">The relative or absolute path of the first file</param>
+        /// <param name="filePath2">The relative or absolute path of the second file</param>
+        [UserCodeMethod]
+        public static void CompareBinaryFiles(string filePath1, string filePath2)

```

Comment:
Change name to ValidateFilesBinaryEqual or similar since it actually does a

Reply:
Done